ctrd/client.go

```diff
@@ -328,3 +346,30 @@ func (c *Client) Cleanup() error {
 
 	return nil
 }
+
+// handleConnectContainerd restore containerd
+func (c *Client) handleConnectContainerd() {
```

Comment:
If containerd process restart, we should also recreate new ctrd client,  WDYT?

Reply:
Maybe client pool need more optimization.

Reply:
```
 gopts := []grpc.DialOption{
        grpc.WithBlock(),
        grpc.WithInsecure(),
        grpc.WithTimeout(60 * time.Second),
        grpc.FailOnNonTempDialError(true),
        grpc.WithBackoffMaxDelay(3 * time.Second),
        grpc.WithDialer(dialer.Dialer),
    }
```
from `containerd.New` grpc connect get a 60s timeout, assumes that we do not consider about containerd killed unexpectly, valid of these connect should be consider.